File: 2023-2024/sourcecode/mod1a/lab2/32StaticFragmentBasicsJava/app/src/main/java/com/example/a32_staticfragmentbasics_java/FragmentB.java
package com.example.a32_staticfragmentbasics_java;

import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.a32_staticfragmentbasics_java.databinding.FragmentABinding;
import com.example.a32_staticfragmentbasics_java.databinding.FragmentBBinding;

public class FragmentB extends Fragment {

    private FragmentBBinding binding;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        binding = FragmentBBinding.inflate(inflater);
        return binding.getRoot();
    }
}

File: 2023-2024/sourcecode/mod1a/lab2/31StatiicFragmentBasicsKotlin/app/src/main/java/com/example/a31_statiicfragmentbasics_kotlin/FragmentB.kt
package com.example.a31_statiicfragmentbasics_kotlin

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.example.a31_statiicfragmentbasics_kotlin.databinding.FragmentABinding
import com.example.a31_statiicfragmentbasics_kotlin.databinding.FragmentBBinding

class FragmentB : Fragment() {

    private lateinit var binding: FragmentBBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = FragmentBBinding.inflate(inflater, container, false)
        return binding.root
    }
}